src/main/java/com/lineate/testyourlexicon/dto/UserDto.java

```diff
@@ -0,0 +1,19 @@
+package com.lineate.testyourlexicon.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class UserDto {
+
+  private final int code = HttpStatus.CREATED.value();
```

Comment:
Statuses are usually not included in successful responses. It is a common practice to include them only for error ones